### ISSUE TITLE
Add back the "gear" icon to fonts or suggest an alternative

### ISSUE THREAD

**Issue by jbarr33** [OfficeDev/office-ui-fabric-core#921]
The gear icon has been fairly standard for invoking user options/settings and it is still in use in Office 365.

I think it was removed from the current icon set or at least I can't find it. Can it be added back to the new icons or can someone suggest a new standard if it has changed?

Thanks.

**Comment by Danny-Engelman** [comment]
its called ``ms--Settings``

**Comment by jbarr33** [comment]
Thanks a lot @Danny-Engelman.  I guessed I missed that. The name changing is a little hard to follow!

**Comment by Danny-Engelman** [comment]
This is my ES6 playground so only works in Chrome (and maybe FF)
https://365csi.nl/toolkit/iconator.html
